recipe/meta.yaml

```diff
@@ -33,14 +33,15 @@ requirements:
     - setuptools
     - setuptools_scm
     - wheel
+    - tomli  # [py<311]
```

Comment:
Should be `toml`, not `tomli`, see https://github.com/zarr-developers/numcodecs/blob/350cbd1eb6751cb32e66aa5a8bc4002077083a83/pyproject.toml#L4. And it's without a constraint